pkg/manager/prow.go

```diff
@@ -7,6 +7,8 @@ import (
 	"encoding/base32"
 	"encoding/json"
 	"fmt"
+	utils "github.com/openshift/ci-chat-bot/pkg"
```

Comment:
Should be named `util` or be unnamed for consistency with the other files in the project.